model/encoder_decoder.py

```diff
@@ -22,7 +22,7 @@ def __init__(
     def forward(self, x, edge_index):
         for index, layer in enumerate(self.layers):
             if index == len(self.layers) - 1:
-                x = layer(x, edge_index)
+                x = layer(x, edge_index).relu()
```

Comment:
in the original model, we didn't have relu on the last layers I believe: https://github.com/pyg-team/pytorch_geometric/blob/master/examples/hetero/hetero_link_pred.py

Reply:
Yes, I know! This is more like a trial. I think it would make more sense to squash the data with a sigmoid at the end rather than a relu.